src/PixiEditor/Models/Dialogs/ExportFileDialog.cs

```diff
@@ -78,11 +78,19 @@ public override bool ShowDialog()
     {
         ExportFilePopup popup = new ExportFilePopup(FileWidth, FileHeight);
         popup.ShowDialog();
+
         if (popup.DialogResult == true)
         {
             FileWidth = popup.SaveWidth;
             FileHeight = popup.SaveHeight;
             FilePath = popup.SavePath;
+
+            if (ChosenFormat.ToString() != "pixi")
```

Comment:
Displaying popup each time user wants to export will be very annoying. This should be saved in editor data (user closed popup) so it only shows once.